templates/vcva_template.json

```diff
@@ -22,11 +22,11 @@
       ],
       "prefix": "__VCENTER_PREFIX_LENGTH__",
       "gateway": "__VCENTER_GATEWAY__",
-      "system_name": "vcva.${domain_name}"
+      "system_name": "__VCENTER_IP__"
     },
     "os": {
       "password": "${vcenter_password}",
-      "ntp_servers": "time.nist.gov",
+      "ntp_servers": "129.6.15.27, 132.163.96.2, 132.163.97.3",
```

Comment:
These are more nist servers:
27.15.6.129.in-addr.arpa domain name pointer time-d-g.nist.gov.
2.96.163.132.in-addr.arpa domain name pointer time-b-b.nist.gov.
3.97.163.132.in-addr.arpa domain name pointer time-c-wwv.nist.gov.

Should we just use time.google.com? https://github.com/packet-labs/google-anthos/issues/119

Reply:
I found that DNS might be an issue. Using 3x NIST servers worked great and will work around DNS issues.
These are IPs spread across 3x regions for NIST so we should be safe from an outage here.

Reply:
The problem that was identified in packet-labs/google-anthos#119 was that vCenter before 6.7u2 will timeout trying to ping the time-server. The nist servers block ICMP.

Since this change is just removing some DNS names, for what are hopefully stable IP addresses (I make assumptions), this change doesn't really bother me.

Reply:
I don't think this will break on vCenter prior to 6.7u2 I'll need to test.

Reply:
@displague I tested these changes with an old version of vCenter 6.7 it worked great. Back in the day (when I was at GE) VMware advised us to use IPs instead of DNS names for NTP when we were experiencing time drift. This was vSphere 5.5 but it looks like IPs are still the less problematic way to go.